robottelo/config/validators.py

```diff
@@ -13,6 +13,7 @@
         Validator('server.hostnames', must_exist=True, is_type_of=list),
         Validator('server.version.release', must_exist=True),
         Validator('server.version.source', must_exist=True),
+        Validator('server.version.rhel_version', cast=int),
```

Comment:
What if we needed a floating version of RHEL somewhere in the tests tomorrow?

e.g :
```
if server.version.rhel_version == 8.7:
    <do this>
else:
    <do that>
```

Reply:
I mean there could be things we might had to handle based on RHEL minor version as well. Doint int casting will remove the minor version from here!